### PR TITLE
Added Bug/Issue and documentation template

### PR DESCRIPTION
**Pull request by DivyanshiNarang**
Resolves issue #192 

Added files for:

* Bug / Issue template
* Documentation template

"Thank you for giving me this opportunity"

Please review it. @PiyushKalyanpy

### CODE REVIEW

File: .github/workflow/ISSUE_TEMPLATE/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this one!
+title: "[DOCS] <description>"
```
Comment (PiyushKalyanpy) [inline .github/workflow/ISSUE_TEMPLATE/documentation.yml:3]:
```suggestion
title: "[DOCS]:"
```
You can remove ` <description>`  from the title and add a colon after brackets.

File: .github/workflow/ISSUE_TEMPLATE/documentation.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
```
Comment (PiyushKalyanpy) [inline .github/workflow/ISSUE_TEMPLATE/documentation.yml:21]:
```suggestion
    id: extra-info
```

File: .github/workflow/ISSUE_TEMPLATE/bug_report.yml
Comment (PiyushKalyanpy) [inline .github/workflow/ISSUE_TEMPLATE/bug_report.yml:1]: We appreciate your hard work, but we have already added a bug report template and we are not aligned with these changes, so you can please remove this file.

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

File: .github/workflow/ISSUE_TEMPLATE/bug_report.yml
Comment (DivyanshiNarang) [reply .github/workflow/ISSUE_TEMPLATE/bug_report.yml:1]: Okay sure.

**Comment by DivyanshiNarang** [comment]
I did the changes you mentioned above. Please review it again. @PiyushKalyanpy

**Review by PiyushKalyanpy — APPROVED** [review]
Thank you @DivyanshiNarang for your valuable contribution 🎉. We would like to have more contributions from you to the Gyanaguru project.

**Pull request merged**